src/main/java/org/fundacionjala/coding/FizzBuzz.java

```diff
@@ -0,0 +1,74 @@
+package org.fundacionjala.coding;
+
+/**
+ * Imagine the scene. You are eleven years old, and in the five minutes
+ * before the end of the lesson, your Maths teacher
+ * decides he should make his class more “fun” by introducing a “game”.
+ * He explains that he is going to point at each
+ * pupil in turn and ask them to say the next number in sequence,
+ * starting from one. The “fun” part is that if the
+ * number is divisible by three, you instead say “Fizz” and if it is
+ * divisible by five you say “Buzz”. So now your
+ * maths teacher is pointing at all of your classmates in turn, and
+ * they happily shout “one!”, “two!”, “Fizz!”, “four!”,
+ * “Buzz!”… until he very deliberately points at you, fixing you with
+ * a steely gaze… time stands still, your mouth dries
+ * up, your palms become sweatier and sweatier until you finally manage to
+ * croak “Fizz!”. Doom is avoided, and the pointing
+ * finger moves on.
+ * <p>
+ * So of course in order to avoid embarassment infront of your whole class,
+ * you have to get the full list printed out so
+ * you know what to say. Your class has about 33 pupils and he might go
+ * round three times before the bell rings for breaktime.
+ * Next maths lesson is on Thursday. Get coding!
+ * <p>
+ * Write a program that prints the numbers from 1 to 100. But for multiples
+ * of three print “Fizz” instead of the number
+ * and for the multiples of five print “Buzz”. For numbers which are
+ * multiples of both three and five print “FizzBuzz “.
+ */
+public final class FizzBuzz {
+    public static final int LIMIT = 101;
+    public static final int THREE = 3;
+    public static final int FIVE = 5;
+    private static final String FIZZ = "Fizz";
+    private static final String BUZZ = "Buzz";
+
+    /**
+     *
+     */
+    public FizzBuzz() {
+
+    }
+
+    /**
+     * Print.
+     */
+    public void printFizzBuzz() {
+        for (int i = 1; i < LIMIT; i++) {
+            System.out.println(processFizzBuzz(i));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)